src/browserbehavior/BrowserBehavior.ts

```diff
@@ -30,7 +30,7 @@ export default interface BrowserBehavior {
   /**
    * Returns whether the browser requires the "playback" latency hint for Web Audio
    */
-  requiresPlaybackLatencyHintForAudioContext(): boolean;
+  requiresPlaybackLatencyHintForAudioContext?(): boolean;
```

Comment:
How about moving it to `ExtendedBrowserBehavior`?

Reply:
Good call, dint knew `ExtendedBrowserBehavior` exists for this purpose :)